draft-ietf-acme-acme.md

```diff
@@ -322,12 +329,18 @@ The use of ACME for other identifiers will require further specification in orde
 to describe how these identifiers are encoded in the protocol and what types of
 validation challenges the server might require.
 
-# Character Encoding
+# Encodings
 
 All requests and responses sent via HTTP by ACME clients, ACME servers, and
 validation servers as well as any inputs for digest computations MUST be encoded
 using the UTF-8 {{!RFC3629}} character set.
 
+Some JSON structures used by ACME include date time values encoded
+as strings.  These values MUST be encoded in the format specified in
+Section 15.0.1.15 of the ECMAScript specification {{ECMAScript}}.
+This format is a simplification of the format defined in
```

Comment:
s/simplification/profile/ maybe?

Reply:
I rolled this back.  It's incompatible with some other libraries (e.g., Go's `time`)

Reply:
I don't think it's accurate that Go's `time` can't handle the date format defined in ECMAScript. Here's an example using a custom layout:

https://play.golang.org/p/6kTORVzXh5e